man/org.freedesktop.systemd1.xml

```diff
@@ -0,0 +1,2040 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd" >
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="org.freedesktop.systemd1" xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>org.freedesktop.systemd1</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>org.freedesktop.systemd1</refentrytitle>
+    <manvolnum>5</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>org.freedesktop.systemd1</refname>
+    <refpurpose>The D-Bus interface of systemd-systemdd</refpurpose>
```

Comment:
```suggestion
    <refpurpose>The D-Bus interface of systemd</refpurpose>
```

Reply:
Ooops, that's my search-and-replace gone wrong.